Tests/TestingApps/PaywallsTester/PaywallsTester/Configuration.swift

```diff
@@ -6,34 +6,64 @@
 //
 
 import Foundation
+import RevenueCat
 
-enum Configuration {
+class Configuration: ObservableObject {
```

Comment:
Nit: make this `final` since it's not meant to be subclassed:
```suggestion
final class Configuration: ObservableObject {
```